### PR TITLE
Begining

### PR DESCRIPTION
**Pull request by MShapira**

### CODE REVIEW

File: src/AppBundle/Controller/ChatPageController.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+class ChatPageController extends Controller
+{
+    public function pageAction()
+    {
+        return new Response(
+            $this->render('ChatPage.html.twig'));
```
Comment (slavchoo) [inline src/AppBundle/Controller/ChatPageController.php:14]: please name all views from a small letter and use low dashes to separate words, for example 'chat_page.html.twig',  also i think the word `page` is no need there.

File: src/AppBundle/Controller/MainPageController.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+class MainPageController extends Controller
+{
+    public function pageAction()
+    {
+        return new Response(
+            $this->render('MainPage.html.twig'));
```
Comment (slavchoo) [inline src/AppBundle/Controller/MainPageController.php:14]: the same as https://github.com/slavchoo/tourreview/pull/1/files#r41352011

File: src/AppBundle/Controller/ChatPageController.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+class ChatPageController extends Controller
+{
+    public function pageAction()
+    {
+        return new Response(
```
Comment (slavchoo) [inline src/AppBundle/Controller/ChatPageController.php:13]: it is no need to wrap `$this->rendor` into `new Response()` you just could use `return $this->render('chat.html.twig');`

File: src/AppBundle/Entity/UserClass.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+namespace AppBundle\Entity;
+
+class UserClass
+{
+    protected $id;
+    protected $username;
+    protected $password;
+    protected $email;
+    protected $first_name;
+    protected $last_name;
```
Comment (slavchoo) [inline src/AppBundle/Entity/UserClass.php:12]: please use camelCase style for class members `protected $lastName;`

File: src/AppBundle/Entity/StoryClass.php [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?php
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+class StoryClass
```
Comment (slavchoo) [inline src/AppBundle/Entity/StoryClass.php:6]: i don't see mapping for your entities

File: web/app.php [hunk 6]
```diff
@@ -4,27 +4,10 @@
 use Symfony\Component\HttpFoundation\Request;
```
Comment (slavchoo) [inline web/app.php:4]: why did you change this file?

Comment (MShapira) [inline web/app.php:4]: I've seen such example in some tutorial. Is it wrong?
